js/stronghold.js

```diff
@@ -0,0 +1,528 @@
+                    ],
+                    'patch': [
+                        'identities',
+                    ],
+                    'put': [
+                        'iam/credentials/{credentialId}',
+                    ],
+                    'delete': [
+                        'iam/credentials/{credentialId}',
+                    ],
+                },
+                'private': {
+                    'get': [
+                        'venues',
+                        'venues/{venueId}/accounts/{accountId}',
+                        'venues/{venueId}/custody/accounts/{accountId}/payment/{paymentId}',
+                        'venues/{venueId}/accounts/{accountId}/orders',
+                        'venues/{venueId}/accounts/{accountId}/trades',
+                        'venues/{venueId}/accounts/{accountId}/transactions',
+                    ],
+                    'post': [
+                        'venues/{venueId}/accounts',
+                        'venues/{venueId}/accounts/{accountId}/orders',
+                        'venues/{venueId}/accounts/{accountId}/deposit',
+                        'venues/{venueId}/accounts/{accountId}/withdrawal',
+                        'venues/{venueId}/custody/accounts/{accountId}/payment',
+                        'venues/{venueId}/custody/accounts/{accountId}/payment/{paymentId}/stop',
+                        'venues/{venueId}/custody/accounts/{accountId}/operations/{operationId}/signatures',
+                        'venues/{venueId}/anchor/withdrawal',
+                        'venues/{venueId}/testing/friendbot',
+                    ],
+                    'delete': [
+                        'venues/{venueId}/accounts/{accountId}/orders/{orderId}',
+                    ],
+                },
+            },
+            'options': {
+                'accountId': undefined,
+                'venueId': 'trade-public',
+                'venues': [ 'trade-public', 'sandbox-public' ],
+                'paymentMethods': {
+                    'ETH': 'ethereum',
+                    'BTC': 'bitcoin',
+                    'XLM': 'stellar',
+                },
+            },
+            'exceptions': {
+                'CREDENTIAL_MISSING': AuthenticationError,
+                'CREDENTIAL_INVALID': AuthenticationError,
+                'CREDENTIAL_REVOKED': AccountSuspended,
+                'CREDENTIAL_NO_IDENTITY': AuthenticationError,
+                'PASSPHRASE_INVALID': AuthenticationError,
+                'SIGNATURE_INVALID': AuthenticationError,
+                'TIME_INVALID': InvalidNonce,
+                'BYPASS_INVALID': AuthenticationError,
+                'INSUFFICIENT_FUNDS': InsufficientFunds,
+            },
+        });
+    }
+
+    async fetchMarkets (params = {}) {
+        const request = {
+            'venueId': this.options['venueId'],
+        };
+        const response = await this.publicGetVenuesVenueIdMarkets (this.extend (request, params));
```

Comment:
Lines 122-125 are now how it should be (how we approach this kinds of logic in other exchanges).